src/shared/Metric/Measurement.cpp

```diff
@@ -57,7 +57,7 @@ std::stringstream& operator<<(std::stringstream& out, const Measurement& measure
         else if (field.second.type() == typeid(float))
             out << boost::any_cast<float>(field.second);
         else if (field.second.type() == typeid(bool))
-            out << (boost::any_cast<bool>(field.second) ? "t" : "f");
```

Comment:
This line should be actually backported to other cores instead. See this commit 03f6a35ce3eb524f81ce84e34f2e948f41ce060d for more information.

Reply:
Ah sorry, missed that. Reverted and put the proper version into vanilla and wotlk PRs.